CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## main / unreleased
 
+* [FEATURE] Make ring distributor gracefulPeriod configurable [#2604](https://github.com/grafana/tempo/pull/2604) (@joffref)
```

Comment:
```suggestion
* [ENHANCEMENT] Make ring distributor gracefulPeriod configurable [#2604](https://github.com/grafana/tempo/pull/2604) (@joffref)
```

It seems more like and enhancement than a feature to me—it's not really adding a new functionality to Tempo.